src/gobble.py

```diff
@@ -64,49 +59,23 @@ def main():
         bus_thread.join()
 
 
-def client_thread(
-    url: str,
-    gtfs_service_date: date,
-    scheduled_trips: pd.DataFrame,
-    scheduled_stop_times: pd.DataFrame,
-    stops: pd.DataFrame,
-):
+def client_thread(url: str):
     logger.info(f"Connecting to {url}...")
     client = sseclient.SSEClient(requests.get(url, headers=HEADERS, stream=True))
 
     current_stop_state: Dict = disk.read_state()
 
-    process_events(client, current_stop_state, gtfs_service_date, scheduled_trips, scheduled_stop_times, stops)
+    process_events(client, current_stop_state)
 
 
-def process_events(
-    client: sseclient.SSEClient,
-    current_stop_state: dict,
-    gtfs_service_date: date,
-    scheduled_trips: pd.DataFrame,
-    scheduled_stop_times: pd.DataFrame,
-    stops: pd.DataFrame,
-):
+def process_events(client: sseclient.SSEClient, current_stop_state):
     for event in client.events():
         try:
             if event.event != "update":
                 continue
 
             update = json.loads(event.data)
-
-            # check for new day
-            updated_at = datetime.fromisoformat(update["attributes"]["updated_at"])
-            service_date = util.service_date(updated_at)
-
-            if gtfs_service_date != service_date:
-                logger.info(
-                    f"New day! Refreshing GTFS bundle from {gtfs_service_date} to {service_date} and clearing state..."
-                )
-                disk.write_state({})
```

Comment:
@hamima-halim @mathcolo realizing I dropped this in `update_current_gtfs_archive_if_necessary`. Do we need this/what does it do with relation to GTFS?

Reply:
nvm, got to the bottom of it in this thread: https://transitmatters.slack.com/archives/GSJ6F35DW/p1705680401311829?thread_ts=1705677890.833879&cid=GSJ6F35DW